pkg/virt-api/webhooks/mutating-webhook/mutators/clone-create-mutator_test.go

```diff
@@ -27,19 +28,19 @@ var _ = Describe("Clone mutating webhook", func() {
 
 		mutator := NewCloneCreateMutator(staticString)
 
-		resp := mutator.Mutate(admissionReview)
-		Expect(resp.Allowed).Should(BeTrue())
+		expectedVirtualMachineCloneSpec := vmClone.Spec.DeepCopy()
+		expectedVirtualMachineCloneSpec.Target = newExpectedTarget(expectedVirtualMachineCloneSpec.Source)
 
-		cloneSpec := &clonev1alpha1.VirtualMachineCloneSpec{}
-		patch := []patch.PatchOperation{
-			{Value: cloneSpec},
-		}
+		expectedJSONPatch, err := expectedJSONPatch(expectedVirtualMachineCloneSpec)
```

Comment:
Same, I think it is valuable to read directly what it is expected.

Reply:
Its a slice of strings, so this part is a bit problematic to do.
This was also done in https://github.com/kubevirt/kubevirt/pull/11561.